### PR TITLE
Fix recurrence part content and tooltip after daylights saving change (T804886, second step)

### PR DESCRIPTION
**Pull request by Yarovaya**

### CODE REVIEW

File: js/ui/scheduler/ui.scheduler.js [hunk 1]
```diff
@@ -1063,18 +1063,21 @@ const Scheduler = Widget.inherit({
         return this._calculateTimezoneByValue(this.option('timeZone'), date);
     },
 
-    _getDaylightOffsetByCustomTimezone: function(startDate, endDate) {
+    _getDaylightOffsetByCommonTimezone: function(startDate, endDate, startDateTimezone) {
```
Comment (Olexi) [inline js/ui/scheduler/ui.scheduler.js]: Why does param `startDateTimezone` needed in this function?

File: testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js [hunk 2]
```diff
@@ -4039,12 +4043,13 @@ QUnit.module('Appointments', () => {
                     startDateExpr: 'startDateCustom',
                     endDateExpr: 'endDateCustom',
                     currentView: 'week',
-                    timeZone: 'Asia/Yekaterinburg',
+                    timeZone: 1,
```
Comment (Olexi) [inline testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js]: According to https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone, it should be `string`

Comment (Yarovaya) [reply testing/tests/DevExpress.ui.widgets.scheduler/integration.appointments.tests.js]: For more comfortable testing, we support an integer value in the code. Look at the calculateTimezoneByValue method.
In the test I need such a timezone, after applying which all the data will be displayed for one day and there will be no DST.

File: js/ui/scheduler/ui.scheduler.subscribes.js [hunk 3]
```diff
@@ -759,7 +772,7 @@ const subscribes = {
         return SchedulerTimezones.getTimezonesIdsByDisplayName(displayName);
     },
 
-    getTargetedAppointmentData: function(appointmentData, appointmentElement) {
+    getTargetedAppointmentData: function(appointmentData, appointmentElement, convertDates) {
```
Comment (Krijovnick) [inline js/ui/scheduler/ui.scheduler.subscribes.js]: The `convertDates` is not necessary

Comment (Yarovaya) [reply js/ui/scheduler/ui.scheduler.subscribes.js]: Yes, indeed, the variable is no longer needed.

**Review by Olexi — DISMISSED** [review]

**Review by Krijovnick — DISMISSED** [review]

**Review by andrewmakarov — DISMISSED** [review]

**Review by andrewmakarov — APPROVED** [review]

**Pull request merged**